README.md

```diff
@@ -89,7 +89,11 @@ describe("user", () => {
 
 ## Usage
 
-- Before running the tests, you must complete the configuration `config.toml`.  
+- Before running the tests, you have to create a config file for xo-test.  
+  ```
+  >  cp sample.config.toml config.toml
```

Comment:
It's not a good idea to create it on local. Better suggest to create it in `~/.config/xo-server-test`.